### DISCUSSION TITLE
when I run the example mnist-keras and run the 'init-workspace', I met some problems, the error message is as follows . Could you give me some help to solve the problem below? thx

### DISCUSSION THREAD

**Discussion by slnebo** [HewlettPackard/swarm-learning#22] [Q&A]
when I run the example mnist-keras and run the 'init-workspace', I met some problems, the error message is as follows . Could you give me some help to solve the problem below? thx.

Waiting for spire server to initialize
Starting Swarm Network node
Error opening Certificate -
139706428171928:error:02001002:system library:fopen:No such file or directory:bss_file.c:398:fopen('-','r')
139706428171928:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:400:
unable to load certificate
Error: spiffeid: invalid scheme
run-sn: cannot generate join token
Swarm Network node failed to start.

**Comment by RadhakrishnaJ** [comment]
It seems your license server not running in your system.  Refer below instructions. 
[Installing licenses and starting license server](https://github.com/HewlettPackard/swarm-learning/blob/master/docs/setup.md#installing-licenses-and-starting-license-server)

Please refer below issue, it has instructions to get license server.
[Issue#5](https://github.com/HewlettPackard/swarm-learning/issues/5)

**Comment by slnebo** [comment]
I have already installed the license and started the license sever, but it didnt work. The error message didnt change:
the error message as follow:
Waiting for spire server to initialize
Starting Swarm Network node
Error opening Certificate -
139706428171928:error:02001002:system library:fopen:No such file or directory:bss_file.c:398:fopen('-','r')
139706428171928:error:20074002:BIO routines:FILE_CTRL:system lib:bss_file.c:400:
unable to load certificate
Error: spiffeid: invalid scheme
run-sn: cannot generate join token
Swarm Network node failed to start.







As follow are my license sever web pages.
I find that in the license management the information of licenses could be shown, while&nbsp; in the License Usage page, there is nothing.
Is this important? And is my license sever setting correct?

















------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HewlettPackard/swarm-learning"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年7月12日(星期一) 中午1:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HewlettPackard/swarm-learning] when I run the example mnist-keras and run the 'init-workspace', I met some problems, the error message is as follows . Could you give me some help to solve the problem below? thx (#22)





 
It seems your license server not running in your system.  Refer below instructions.
 Installing licenses and starting license server
 
Please refer below issue, it has instructions to get license server.
 Issue#5
